repos/system_upgrade/el7toel8/actors/inhibitwhenluks/tests/test_inhibitwhenluks.py

```diff
@@ -19,4 +20,4 @@ def test_actor_without_luks(current_actor_context):
 
     current_actor_context.feed(StorageInfo(lsblk=without_luks))
     current_actor_context.run()
-    assert not current_actor_context.consume(Inhibitor)
+    assert not current_actor_context.consume(Report)
```

Comment:
again - Report != Inhibitor. Check flags.

Reply:
It is stated in this PR description as a known issue.